Resets crosshair to blank when shifting to spectate mode

Vhostym opened the pull request:
Now when you change into spectate mode the crosshair will shift from whatever portal combination to blank. The same will happen when you shift back.

Vhostym committed:
Resets crosshair to blank when shifting to spectate mode

ambid17 commented on Assets/Scripts/UI/InGameUI/InGameUI.cs:
```diff
@@ -48,6 +47,7 @@ private void Start()
         playerMovement = GetComponentInParent<PlayerMovement>();
         speedBar = GetComponentInChildren<SpeedSlider>();
         ghostRun = GetComponentInParent<PlayerGhostRun>();
+        playerCrosshair = FindObjectOfType<Crosshair>();
```
GetComponentInParent

ambid17 replied:
always try to find a way to not use the "find" methods, they are super slow

Vhostym committed:
Improvements to code

Improved efficiency of finding the crosshair object. Also fixed issue where ghost crosshair was white instead of orange.
